third_party/python/BUILD

```diff
@@ -56,18 +59,15 @@ python_wheel(
     version = "3.10.0.2",
 )
 
-python_wheel(
+pip_library(
```

Comment:
A lot of them had to be changed to pip_library to avoid circular dependencies with the wheel_resolver tool, but actually this one was fine as a python_wheel